expression/explain.go

```diff
@@ -60,12 +61,28 @@ func (expr *Constant) format(dt types.Datum) string {
 }
 
 // ExplainExpressionList generates explain information for a list of expressions.
-func ExplainExpressionList(exprs []Expression) []byte {
+// needSort will be true if the exprs should keep its order. In some scenarios, the expr's order may not be valid
```

Comment:
why the order of the `exprs` may not be valid?

Reply:
Uh, I think `stable` is a better word. By the way, this function only used when we `explain` a sql, and the performance doesn't matter, I think we can force to sort the `exprs` to simplify the function signature, and this function can be renamed to `SortAndExplainExpressionList`.